app/src/main/kotlin/no/nav/aap/oppgave/oppdater/OppdaterOppgaveService.kt

```diff
@@ -270,6 +274,19 @@ class OppdaterOppgaveService(
         }
     }
 
+    private fun hentVeilederSykefraværoppfølging(personIdent: String?): String? =
```

Comment:
Ikke helt bestemt meg hva jeg tenker om signaturer som tar optional og gir optional 😅 

Ikke en suggestion, bare penere signatur?
```suggestion
    private fun hentVeilederSykefraværoppfølging(personIdent: String): String =
```

Reply:
Enig, fikser den 👍